README.md

```diff
@@ -9,7 +9,11 @@ npm install --global yarn
 Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
 yarn install
 ```
-
+## Googel Maps API Setup
+```bash
+npm install @react-google-maps/api
+npm install @types/react @types/react-dom
+```
```

Comment:
these commands wont be needed since if you run `yarn install`  itll just download all of these automatically and update the yarn.lock file.

in your branch just be sure to run these commands so it saves it in the lock file
`yarn add @react-google-maps/api --dev`
`yarn add @types/react @types/react-dom --dev`

Reply:
Removed and updated lock file